app/controllers/RelayerController.js

```diff
@@ -0,0 +1,91 @@
+module.exports = (osseus) => {
+  return {
+    transmit: async (req, res, next) => {
+      try {
+        osseus.logger.debug(`RelayerController.transmit ${JSON.stringify(req.body, null, '\t')}`)
+
+        // delegate account
+        const delegatePrivateKey = osseus.config.sender_pkey
+        if (!delegatePrivateKey) {
+          osseus.logger.error(`SENDER_PKEY missing in config`)
+          return next(`Internal error`)
+        }
+        osseus.logger.debug(`delegatePrivateKey: ${delegatePrivateKey}`)
+        const delegate = osseus.utils.privateKeyToAddress(delegatePrivateKey)
+        osseus.logger.debug(`delegate: ${delegate}`)
+        const delegateNonce = await osseus.web3.eth.getTransactionCount(delegate)
+        osseus.logger.debug(`delegateNonce: ${delegateNonce}`)
+
+        // validations
+        if (!osseus.web3.utils.isAddress(req.body.token_address)) {
+          return next('Validation error - token_address is invalid')
+        }
+        if (!osseus.web3.utils.isAddress(req.body.from_account)) {
+          return next('Validation error - from_account is invalid')
+        }
+        if (!osseus.web3.utils.isAddress(req.body.to_account)) {
+          return next('Validation error - to_account is invalid')
+        }
+        req.body.amount_wei = osseus.web3.utils.toBN(req.body.amount_wei)
+
+        // init token
+        const Token = new osseus.web3.eth.Contract(osseus.config.erc677_bridge_token_abi, req.body.token_address)
+
+        // validate `from_address` nonce
+        const nonce = await Token.methods.getNextNonceForAddress(req.body.from_account).call()
```

Comment:
@bejavu @leonprou 
Not sure we want/need this (especially if we change it to be timestamps), WDYT?